Makefile

```diff
@@ -571,7 +570,7 @@ ginkgo-remote:
 
 .PHONY: testbindings
 testbindings: .install.ginkgo
-	ACK_GINKGO_RC=true $(GINKGO) -v $(TESTFLAGS) -tags "$(TAGS) remote" $(GINKGOTIMEOUT) -progress -trace -noColor -debug -timeout 30m  -v -r ./pkg/bindings/test
+	$(GINKGO) -v $(TESTFLAGS) --tags "$(TAGS) remote" $(GINKGOTIMEOUT) --trace --no-color --timeout 30m  -v -r ./pkg/bindings/test
```

Comment:
There's a `$(GINKGOTIMEOUT)` specified here in addition to a `--timeout 30m`, not sure if that was intentional / doing some tricky thing with ginkgo or not.  Consider removing one or the other or adding a comment detailing why there are two timeout options.